### PR TITLE
ASICs, Bitcoin & SHA256: \cite{ASICSmithMJS} \cite{Courtois2014}

### PR DESCRIPTION
**Pull request by jamesray1**

### CODE REVIEW

**Comment by pirapira** [comment]
I'll first look at those without conflicts.

**Comment by jamesray1** [comment]
@pirapira

File: Biblio.bib [hunk 1]
```diff
@@ -1,3 +1,32 @@
...
+	url="https://doi.org/10.1007/978-3-662-44893-9_12"
+}
+
+@book{ASICSmithMJS,
+	title = {{Application-Specific Integrated Circuits}},
+	author = "Smith, Michael John Sebastian",
+	publisher = "Addison-Wesley",
+	year = "1997",
+	ISBN = "0201500221",
+	note = "Available here: \url{https://www.slideshare.net/nhKhanhNguyn/michael-john-sebastian-smith-applicationspecific-integrated-circuitsaddisonwesley-professional-1997-1}, which is not able to be archived by the Wayback Machine, but is available here: \url{https://drive.google.com/open?id=0B6erNjejWCBwbTc1RmFLV1BlZlE}. A preview is also available at \url{https://www.amazon.com/Application-Specific-Integrated-Circuits-Michael-Smith/dp/0321602757}, with the archive:
```
Comment (pirapira) [inline Biblio.bib]: I cannot help aiding copyright infringements.

Comment (jamesray1) [reply Biblio.bib]: Fixed.

**Comment by pirapira** [comment]
Will you remove it from the commit history, too?

**Comment by jamesray1** [comment]
#629

**Pull request closed without merging**